From db30f3b2a194c3c604d06253cf5c55bc7e6abbc8 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 2 Feb 2022 17:24:07 +0100
Subject: [PATCH] treewide: MOX: rename the target to more appropriate one

It was changed that the MOX is now the same as Turris Omnia
---
 hardware/mox/mox-dtb/Makefile                  | 2 +-
 hardware/mox/mox-firmware/Makefile             | 4 ++--
 hardware/mox/mox-otp/Makefile                  | 2 +-
 hardware/mox/mox-pkcs11/Makefile               | 2 +-
 hardware/mox/mox-support/Makefile              | 6 +++---
 hardware/mox/mox-uboot/Makefile                | 2 +-
 hardware/rescue-image/Makefile                 | 4 ++--
 web/foris-controller/foris-controller/Makefile | 2 +-
 8 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/hardware/mox/mox-dtb/Makefile b/hardware/mox/mox-dtb/Makefile
index 4f70dc541..197424bcc 100644
--- a/hardware/mox/mox-dtb/Makefile
+++ b/hardware/mox/mox-dtb/Makefile
@@ -19,7 +19,7 @@ include $(INCLUDE_DIR)/package.mk
 
 define Package/mox-dtb
   TITLE:=Device tree for Turris MOX
-  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic-mox
+  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox
 endef
 
 define Package/mox-dtb/description
diff --git a/hardware/mox/mox-firmware/Makefile b/hardware/mox/mox-firmware/Makefile
index 19a403757..759bc32a4 100644
--- a/hardware/mox/mox-firmware/Makefile
+++ b/hardware/mox/mox-firmware/Makefile
@@ -17,7 +17,7 @@ include $(INCLUDE_DIR)/package.mk
 
 define Package/turris-mox-firmware
   TITLE:=Firmware for Turris MOX
-  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic-mox
+  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox
   PROVIDES:=turris-nor-update-firmware
 endef
 
@@ -27,7 +27,7 @@ endef
 
 define Package/turris-mox-devel-firmware
   TITLE:=Data files for nor-update (development version)
-  DEPENDS:=+turris-nor-update +rescue-image +u-boot-mox @TARGET_mvebu_cortexa53_DEVICE_cznic-mox
+  DEPENDS:=+turris-nor-update +rescue-image +u-boot-mox @TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox
   PROVIDES:=turris-nor-update-devel-firmware
   VERSION:=0.1
   RELEASE:=1
diff --git a/hardware/mox/mox-otp/Makefile b/hardware/mox/mox-otp/Makefile
index 6e6c7d394..8c33f7054 100644
--- a/hardware/mox/mox-otp/Makefile
+++ b/hardware/mox/mox-otp/Makefile
@@ -28,7 +28,7 @@ define Package/mox-otp
   URL:=https://gitlab.nic.cz/turris/mox-otp
   PROVIDES:=turris-otp
   DEPENDS:=+python3-light
-  DEPENDS+=@TARGET_mvebu_cortexa53_DEVICE_cznic-mox
+  DEPENDS+=@TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox
   VARIANT:=python3
 endef
 
diff --git a/hardware/mox/mox-pkcs11/Makefile b/hardware/mox/mox-pkcs11/Makefile
index 3bb5b5dee..91295b5b1 100644
--- a/hardware/mox/mox-pkcs11/Makefile
+++ b/hardware/mox/mox-pkcs11/Makefile
@@ -23,7 +23,7 @@ include $(INCLUDE_DIR)/package.mk
 
 define Package/mox-pkcs11
   TITLE:=MOX PKCS11
-  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic-mox +libopenssl
+  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox +libopenssl
 endef
 
 define Package/mox-pkcs11/description
diff --git a/hardware/mox/mox-support/Makefile b/hardware/mox/mox-support/Makefile
index f38fdd781..24f407d58 100644
--- a/hardware/mox/mox-support/Makefile
+++ b/hardware/mox/mox-support/Makefile
@@ -16,18 +16,18 @@ include $(INCLUDE_DIR)/package.mk
 
 define Package/mox-generic-support
   TITLE:=Generic MOX support package
-  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic-mox \
+  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox \
     +mtd +kmod-gpio-button-hotplug +mox-dtb
 endef
 
 define Package/mox-support
   TITLE:=MOX support package
-  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic-mox +mox-generic-support
+  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox +mox-generic-support
 endef
 
 define Package/shield-support
   TITLE:=Shield support package
-  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic-mox +mox-generic-support \
+  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox +mox-generic-support \
 	  +pkglists +turris-nor-update +turris-mox-firmware
   PROVIDES:=mox-support
   CONFLICTS:=mox-support
diff --git a/hardware/mox/mox-uboot/Makefile b/hardware/mox/mox-uboot/Makefile
index d6ebe5c7e..1625e4f24 100644
--- a/hardware/mox/mox-uboot/Makefile
+++ b/hardware/mox/mox-uboot/Makefile
@@ -24,7 +24,7 @@ include $(INCLUDE_DIR)/package.mk
 
 define Package/mox-uboot
   TITLE:=U-boot for Turris MOX
-  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic-mox
+  DEPENDS:=@TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox
   PROVIDES:=u-boot-mox
 endef
 
diff --git a/hardware/rescue-image/Makefile b/hardware/rescue-image/Makefile
index d41b79480..b845c66b1 100644
--- a/hardware/rescue-image/Makefile
+++ b/hardware/rescue-image/Makefile
@@ -31,7 +31,7 @@ include $(INCLUDE_DIR)/quilt.mk
 include $(INCLUDE_DIR)/package.mk
 
 PATCH_DIR := $(PLATFORM_DIR)/patches$(if $(wildcard $(PLATFORM_DIR)/patches-$(KERNEL_PATCHVER)),-$(KERNEL_PATCHVER))
-ifeq ($(CONFIG_TARGET_mvebu_cortexa53_DEVICE_cznic-mox),y)
+ifeq ($(CONFIG_TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox),y)
 DEVICE := mox
 else ifeq ($(CONFIG_TARGET_mvebu_cortexa9_DEVICE_cznic_turris-omnia),y)
 DEVICE := omnia
@@ -42,7 +42,7 @@ kernel_files=$(TOPDIR)/target/linux/$(BOARD)/files/. $(TOPDIR)/target/linux/gene
 
 define Package/rescue-image
 	TITLE:=rescue-image
-	DEPENDS:=@(TARGET_mvebu_cortexa53_DEVICE_cznic-mox||TARGET_mvebu_cortexa9_DEVICE_cznic_turris-omnia)
+	DEPENDS:=@(TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox||TARGET_mvebu_cortexa9_DEVICE_cznic_turris-omnia)
 endef
 
 define Build/Prepare
diff --git a/web/foris-controller/foris-controller/Makefile b/web/foris-controller/foris-controller/Makefile
index 9bfd6c324..e1c148ad8 100644
--- a/web/foris-controller/foris-controller/Makefile
+++ b/web/foris-controller/foris-controller/Makefile
@@ -63,7 +63,7 @@ define Py3Package/foris-controller-app/install
 	$(INSTALL_DIR) $(1)/etc/config/
 	$(INSTALL_CONF) ./files/foris-controller.conf $(1)/etc/config/foris-controller
 	touch $(1)/etc/config/foris
-ifeq ($(CONFIG_TARGET_mvebu_cortexa53_DEVICE_cznic-mox),y)
+ifeq ($(CONFIG_TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox),y)
 	sed -i 's|#\(option single true\)|\1|' $(1)/etc/config/foris-controller
 endif
 	$(INSTALL_CONF) ./files/foris-controller-subordinates.conf $(1)/etc/config/foris-controller-subordinates
-- 
2.32.0

